nixos/modules/services/web-apps/firefly-iii-data-importer.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
```

Comment:
You might want to add the module to `nixos/modules/module-list.nix` :)

Reply:
Yup forgot about that indeed, thanks